lib/main/STM32H7/Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal_sd.h

```diff
@@ -121,6 +121,10 @@ typedef struct __SD_HandleTypeDef
 typedef struct
 #endif /* USE_HAL_SD_REGISTER_CALLBACKS */
 {
+  uint32_t                     CSD[4];           /*!< SD card specific data table         */
```

Comment:
Should this have gone into #10246?

Reply:
This change shouldn't be needed so I'm dropping it. This seems to be necessary to get the code to work with the .bss section being put into DTCM. Bootloader seems to hang. It works fine without this change if .bss is put in D2_RAM. There's not enough space in RAM, and it would seem sensible to put .bss in DTCM.